modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java

```diff
@@ -605,6 +605,10 @@ private DDMForm _getDDMForm(ObjectLayoutTab objectLayoutTab)
 					continue;
 				}
 
+				if (objectField.isSystem()) {
```

Comment:
Hey @mateussgn,
What do you think about joining the two conditions? I think it would be better since the action is the same.